code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -74,7 +74,10 @@
 
 /datum/symptom/proc/generate_threshold_desc()
 	return
-
+	
+/datum/symptom/proc/OnDeath(datum/disease/advance/A)
+	return !neutered

```

Comment:
I don't understand viro code very well. Why are you returning `!neutered`? Shouldn't it just be `neutered`?

Reply:
I am unsure. the code i used here is some shit from an older /tg/ symptom pr that went stale for having "fetish content" (fart symptoms). As i understand, it only returns if the symptom is not neutered.

Reply:
No, it returns the _opposite_ of the neutered value. So if neutered is false, it returns true. And vice-versa.